test/browser_tests.py

```diff
@@ -14,7 +14,8 @@ class BrowserTests(Common):
     DEFAULT_PDF2HTMLEX_ARGS = [
         '--fit-width', 800,
         '--last-page', 1,
-        '--embed', 'fi', # avoid base64 to make it faster
+        '--embed', 'FI', # with 'fi', test suite is currently broken (some directory interference)
+        # avoid base64 to make it faster
```

Comment:
By using `FI`, base64 _is_ used, so this line is no longer true. Please remove it.